src/rendition-mixin.js

```diff
@@ -30,8 +30,9 @@ const enableFunction = (loader, playlistUri, changePlaylistFn, enable) => {
   if (enable !== currentlyEnabled && !blacklisted) {
     // Ensure the outside world knows about our changes
     changePlaylistFn();
+    // fired when a rendition is enabled/disabled in HLS
+    loader.trigger('hlsrenditiondisabled');
```

Comment:
since we control this event and its being triggered on the `PlaylistLoader` and not the `tech`, we don't need to have the `hls` prefix. But the `usage` event fired on the `tech` should still have the `hls-` prefix